wp-job-manager-functions.php

```diff
@@ -1712,3 +1717,80 @@ function job_manager_get_salary_unit_options( $include_empty = true ) {
 	 */
 	return apply_filters( 'job_manager_get_salary_unit_options', $options, $include_empty );
 }
+
+/**
+ * Checks if the job expiry can be extended.
+ * This is true if the job is public, the option to extend is set and the job expires within 5 days.
+ *
+ * @since $$next_version$$
+ *
+ * @param int|WP_Post $job The job or job ID.
+ *
+ * @return bool
+ */
+function job_manager_job_can_be_renewed( $job ) {
```

Comment:
As you introduced a new class, maybe it makes sense to move these global functions inside the class?